### PR TITLE
6 get comments

### PR DESCRIPTION
**Pull request by YShamm**
Task 6

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -97,3 +97,41 @@ describe("GET /api/articles/:article_id, gets an article by its id", () => {
...
+        expect(comments.length).toBe(2);
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+
+        comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at");
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("body");
+          expect(comment).toHaveProperty("article_id");
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:118]: You can be more specific here.What article id do you expect these comments to have?

File: __tests__/app.test.js [hunk 2]
```diff
@@ -97,3 +97,41 @@ describe("GET /api/articles/:article_id, gets an article by its id", () => {
...
+  });
+  test("status 404; id not found - id type correct but does not exist", () => {
+    return request(app)
+      .get("/api/articles/9000/comments")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("article id not found");
+      });
+  });
+  test("status 400; invalid id type - id type incorrect", () => {
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:150]: Nice error tests here👏

File: __tests__/app.test.js [hunk 3]
```diff
@@ -97,3 +97,41 @@ describe("GET /api/articles/:article_id, gets an article by its id", () => {
...
+    return request(app)
+      .get("/api/articles/three/comments")
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("invalid id");
+      });
+  });
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:157]: What if the article has no comments? This is a test case to consider

**Review by Beth-McCormack — COMMENTED** [review]
Nice work! I've left a few comments for you🙂

File: __tests__/app.test.js (hunk 1 above)
Comment (two-triangles) [reply __tests__/app.test.js:118]: To add one more thing to this, you could be testing that the typeof the property values is what you expect it to be! Right now you are only testing that your comments contain the property keys

**Pull request merged**